asdf/generic_io.py

```diff
@@ -112,7 +111,7 @@ def _array_tofile(fd, write, array):
         if fd is None or array.nbytes >= OSX_WRITE_LIMIT and array.nbytes % 4096 == 0:
             return _array_tofile_chunked(write, array, OSX_WRITE_LIMIT)
         return _array_tofile_simple(fd, write, array)
-elif sys.platform.startswith('win'):  # pragma: no cover
+elif sys.platform.startswith('win'):
     def _array_tofile(fd, write, array):
         WIN_WRITE_LIMIT = 2 ** 30
         return _array_tofile_chunked(write, array, WIN_WRITE_LIMIT)
```

Comment:
`OSX_WRITE_LIMIT` and `WIN_WRITE_LIMIT` are not really global variables here, so should probably be lower case.  Or you could make them global variables.  They're identical.

Reply:
That is a bit strange.  I'll move those outside the functions.

Reply:
Actually, I'm going to tackle this in a separate PR.  These workarounds seem to no longer be necessary, but this PR will be included in an asdf 2.7.3 and I'd rather drop them in 2.8.

Reply:
https://github.com/asdf-format/asdf/issues/932